samples/ldscripts/armv6m_soft_nofp.ld

```diff
@@ -1,7 +1,7 @@
-__flash = 0x00000000;   /* starting address of flash */
```

Comment:
This file is actually part of a sample used for educational purposes so please restore the comments

Reply:
Was not aware bout that. Comments are back.